stdlib/LibGit2_jll/src/LibGit2_jll.jl

```diff
@@ -21,9 +21,9 @@ libgit2_path = ""
 if Sys.iswindows()
     const libgit2 = "libgit2.dll"
 elseif Sys.isapple()
-    const libgit2 = "@rpath/libgit2.1.5.dylib"
+    const libgit2 = "@rpath/libgit2.1.6.dylib"
 else
-    const libgit2 = "libgit2.so.1.5"
+    const libgit2 = "libgit2.so.1.6"
```

Comment:
Sounds like libgit2 v1.6 has still soversion 1.5.  Not sure that was done on purpose though, although it'd be great if they didn't break the  ABI again.

Reply:
Might have not been done on purpose, given the fact now there's v1.6.1 which includes this: https://github.com/libgit2/libgit2/commit/8a871d13b7f4e186b8ad943ae5a7fcf30be52e67

Reply:
And [v1.6.0](https://github.com/libgit2/libgit2/releases/tag/v1.6.0) was retracted

Reply:
It looks like the build product is still called `libgit2.so.1.5`
https://github.com/JuliaBinaryWrappers/LibGit2_jll.jl/releases/tag/LibGit2-v1.6.0%2B0

Reply:
Yeah, they tagged 1.6.0 ***before*** merging https://github.com/libgit2/libgit2/pull/6501, the soversion was not bumped in v1.6.0.  This has been rectified in v1.6.1.